problem-16/src/app/pipe/ModifyComment.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'ModifyComment',
+    standalone: true
+
+})
+export class ModifyComment implements PipeTransform {
```

Comment:
ModifyComment -> modify-comment

Reply:
changed file and selector to modify-comment